modules/admin_manual/pages/release_notes.adoc

```diff
@@ -19,6 +20,160 @@
+
+=== PHP 5.6/7.0 Deprecation
+
+Following up the https://doc.owncloud.com/server/admin_manual/release_notes.html#php-5-6-deprecation[PHP 5.6/7.0 deprecation notice in the ownCloud Server 10.0.8 releases] ownCloud Server 10.2 *does not support PHP 5.6* and some apps no longer support older PHP versions. 
+Additionally, the next, minor, version of ownCloud Server will support PHP 7.3.
+
+If you're still running PHP 5.6, **you must upgrade to PHP 7 before upgrading to ownCloud Server 10.2**. 
+Please be aware that apps that do not support outdated PHP versions will not upgrade.
+
+TIP: See the xref:installation/system_requirements.adoc#officially-recommended-supported-options[system requirements in the ownCloud documentation]. 
+
+To allow for additional upgrade time, version 10.2 still supports PHP 7.0, because some of the major Linux distributions continue to support it. 
+However, support for PHP 7.0 will be discontinued in an upcoming version of ownCloud 10, to enhance both security and performance. 
+To prepare for this change, we strongly encourage you to begin planning an upgrade as soon as possible.
+
+=== Advanced Sharing Permissions
+
+The new server version https://github.com/owncloud/core/issues/34951[introduces the means] for extensions to implement additional, advanced permissions for user and group sharing. This feature increases sharing flexibility and opens the doors for extension developers to introduce new functionality based on sharing permissions.
+
+Especially, considering collaborative editing solutions, this addition provides the foundation for mode-based document sharing, such as "view-only", "comments-only" or "enforce change tracking". In the future, such advanced permissions should significantly improve the security as well as the usability of review processes, working on Office documents collaboratively, or exchanging information securely.
+
+Based on the new capabilities a set of features has been developed together with Collabora Online, called _Secure View_. Secure View is designed to enable information distribution processes for sensitive data, meaning that information can be provided securely yet can — *under no circumstances* — leave the platform.
+
+Practically, it enables users to share documents (such as docx, xlsx, pptx, and PDF files) in such a way that the recipient can't edit, download, copy and paste, nor print them. 
+Additional protection for screenshots and photos is provided by watermarks which display user information. 
+What's more, users can decide to allow printing and exporting of documents protected by watermarks as well.
+
+=== More Granular Permissions for Public Links on Folders
+
+With Server 10.2 a new permission for public links on folders is available for users. 
+It lets recipients view, download, and upload documents, yet prevents them from make changes (e.g., rename, move, collaborative editing). 
```

Comment:
"prevents them from **make** changes" => is that correct?

Reply:
s/make/making/

Reply:
Thanks. Will fix.